java/arcs/core/data/proto/policy.proto

```diff
@@ -0,0 +1,107 @@
+syntax = "proto3";
+
+package arcs;
+
+option java_package = "arcs.core.data.proto";
+option java_multiple_files = true;
+
+// Defines a data usage policy.
+message PolicyProto {
+  // Policy name.
+  string name = 1;
+
+  // Targets governed by the policy.
+  repeated PolicyTargetProto targets = 2;
+
+  // Additional config metadata for the policy.
+  repeated PolicyConfigProto configs = 3;
+
+  // Human readable description for the policy.
+  string description = 4;
+
+  enum EgressType {
+    EGRESS_TYPE_UNSPECIFIED = 0;
+    LOGGING = 1;
+    FEDERATED_ANALYTICS = 2;
+  }
+  // Egress type permitted by the policy.
+  EgressType egress_type = 5;
+
+  // Whether user consent is required before using the policy.
+  bool user_consent_required = 6;
+
+  enum ApprovalCondition {
+    APPROVAL_CONDITION_UNSPECIFIED = 0;
+    PRE_APPROVED = 1;
+    APPROVAL_REQUIRED = 2;
+  }
+  // Approval condition for the policy.
+  ApprovalCondition approval_condition = 7;
+}
+
+message PolicyTargetProto {
+  // ID of the store targeted by the policy.
+  string store_id = 1;
+
+  // Valid retention options for storing the target data.
+  repeated PolicyRetentionProto retentions = 2;
+
+  // Allowed usages for all fields in this target type.
+  repeated PolicyFieldProto fields = 3;
+}
+
+message PolicyFieldProto {
+  // Field name.
+  string name = 1;
+
+  enum UsageType {
+    USAGE_TYPE_UNSPECIFIED = 0;
```

Comment:
This only makes sense if you use this enum in non repeated fields, as this will be the default value if value is not set. You currently have it as a repeated field, so this should not show up. It's ok to leave it if you want to be future proof against usage in optional fields, up to you.

Reply:
I'm just trying to follow best practices and set a good example ;-) 

go/protodosdonts#do-include-an-unspecified-value-in-an-enum